contrib/packs/actions/pack_mgmt/setup_virtualenv.py

```diff
@@ -98,9 +98,15 @@ def _setup_pack_virtualenv(self, pack_name):
                           (pack_name, virtualenv_path))
 
     def _create_virtualenv(self, virtualenv_path):
-        self.logger.debug('Creating virtualenv in "%s"' % (virtualenv_path))
```

Comment:
Yeah, I guess so, but "packs" pack is a core pack so imo this is fine here. Among other things, actions in this pack also already access other config options so they can connect to the DB to remove content...